src/components/VImg/VImg.vue

```diff
@@ -168,10 +170,15 @@ export default {
 
 .vts-img__placeholder {
   position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
   overflow: hidden;
 }
 
 .vts-img__placeholder img {
+  width: 100%;
```

Comment:
I'm sure most folks would want this sort of style, but I'm trying to avoid making assumptions on what people want, and to avoid any unnecessary styles as much as possible. Is there a reason this needs to be here?

Reply:
you are right, not everybody wants the placeholder-image to stretch full with of the container.
I removed the fixed style and applied the img-class-prop to the preview-img as well, that solves the problem of having a flickr in size from preview to full image.